doc/src/content/xdocs/spec.xml

```diff
@@ -328,15 +328,20 @@
     <section>
       <title>Data Serialization</title>
 
-      <p>Avro data is always serialized with its schema.  Files that
-	store Avro data should always also include the schema for that
-	data in the same file.  Avro-based remote procedure call (RPC)
-	systems must also guarantee that remote recipients of data
-	have a copy of the schema used to write that data.</p>
-
-      <p>Because the schema used to write data is always available
-	when the data is read, Avro data itself is not tagged with
-	type information.  The schema is required to parse data.</p>
+      <p>Avro data does not include type information.
+      The binary encoding does not include field names, either.  The resulting
+      tradeoff is that the serialized data is small, but the original schema of
+      the writer is always required in order to read Avro data correctly.</p>
+
+      <p>Therefore, files that store Avro data should always include the
+      schema for that data in the same file.  Avro-based remote procedure
+      call (RPC) systems must also guarantee that remote recipients of data
+      have a copy of the schema used to write that data.  In general, any
+      reader of Avro data must a schema that is structurally identical (as
```

Comment:
The parsing canonical form spec doesn't define structurally identical schemas. There are schemas that are structurally identical, but don't have the same parsing canonical form. I think it might be a good idea to just remove this sentence entirely. Do we need to state that a structurally identical schema can be used to read a file? It seems like that's goign to be more confusing than its worth.